proto/composable/xcvm/v1beta1/ethereum.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+package composable.xcvm.v1beta1;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "x/xcvm/types";
+
+message ClientState {
+  LightClientState inner = 1;
+  optional uint64 frozen_height_revision_height = 2;
+  optional uint64 frozen_height_revision_number = 3;
+  uint64 latest_height = 4;
+}
+
+message SyncCommittee {
+  repeated bytes public_keys = 1;
+  bytes aggregate_public_key = 2;
+}
+
+message LightClientState {
+  BeaconBlockHeader finalized_header = 1;
+  uint64 latest_finalized_epoch = 2;
+  SyncCommittee current_sync_committee = 3;
+  SyncCommittee next_sync_committee = 4;
+}
+
+// A beacon block header is essentially a beacon block with only a reference to
+// the beacon body as a 32 byte merkle tree root. This type of message is more
+// lightweight than a full beacon block. The message does not contain
+// a validator signature.
+message BeaconBlockHeader {
```

Comment:
We could probably just pull this `BeaconBlockHeader` message directly off of prysm

Reply:
wdym? import the protobuf definition instead of copying?

Reply:
Yeah that's what I was thinking, it's probably not necessary though for the prototype